src/schema/sources.ts

```diff
@@ -1136,6 +1169,12 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
   BaseContext
 >({
   Query: {
+    sourceCategories: async (
+      _,
+      __,
+      ctx: Context,
+    ): Promise<GQLSourceCategory[]> =>
+      ctx.con.getRepository(SourceCategory).find(),
```

Comment:
We should use graphorm to query so any field resolvers also work. Also can we make the query support pagination as other listings?

Reply:
Yeah, we can do that.